lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -334,12 +334,14 @@ def gem_version_compare(first, second)
             Gem::Version.new(first.to_s) <=> Gem::Version.new(second.to_s)
           end
 
-          apipie :method, "Returns content of 'SSL CA file' configured in Settings > Authentication" do
+          apipie :method, "Returns the certificate of Foreman Server CA" do
```

Comment:
It doesn't have to be a single certificate, it can be multiple. The curl manpage states:
>        --cacert <file>
>              (TLS) Tells curl to use the specified certificate file to verify the peer. The file may contain multiple CA certificates. The certificate(s) must be in PEM format.
>              Normally curl is built to use a default file for this, so this option is typically used to alter that default file.

```suggestion
          apipie :method, "Returns the TLS certificate(s) needed to verify a connection to Foreman" do
```

Perhaps the macro should be `foreman_tls_ca_bundle` to reflect it also contains the intermediate certificates.